bundles/gcp-bundle/build.gradle.kts

```diff
@@ -42,6 +42,12 @@ tasks.withType(ShadowJar::class.java) {
   relocate("com.google.common", "org.apache.gravitino.gcp.shaded.com.google.common")
   relocate("com.fasterxml", "org.apache.gravitino.gcp.shaded.com.fasterxml")
   relocate("org.eclipse.jetty", "org.apache.gravitino.gcp.shaded.org.eclipse.jetty")
```

Comment:
why not shade the package in `gcp` module too?

Reply:
Gravitino server will use `gcp-bundle` for Hadoop catalog only. The conflict only exists in Gravitino server now, so do you mean you need to also consider module `gcp` for future use case?

Reply:
OH, seems no requrirement.